plugins/modules/foreman_host.py

```diff
@@ -63,6 +86,25 @@
     hostgroup: my_hostgroup
     state: present
 
+- name: "Create a host with build context"
+  foreman_host:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    name: "new_host"
+    hostgroup: my_hostgroup
+    build: yes
```

Comment:
```suggestion
    build: true
```
(Ansible accepts "yes", but I prefer to have "real" booleans in the examples)

Reply:
sure, it must be homogeneous